meteor/client/styles/settings.scss

```diff
@@ -240,6 +240,19 @@
 	}
 }
 
+.showStyleVariant-invalid-blueprint-configuration {
```

Comment:
That is not something I am used to be using. Do you have a suggestions for how this specific ccs selector would be named with BEM?

Reply:
To make it consistent with other classes in that component, it would probably be something like `.settings-studio-showStyleVariant__header--invalid`. I'm not necessarily a fan of mixing camel case with kebab case, but it is how it is